src/netbase.cpp

```diff
@@ -293,7 +293,7 @@ struct ProxyCredentials
 };
 
 /** Connect using SOCKS5 (as described in RFC1928) */
-bool static Socks5(string strDest, int port, const ProxyCredentials *auth, SOCKET& hSocket)
+bool static Socks5(const std::string& strDest, int port, const ProxyCredentials *auth, SOCKET& hSocket)
```

Comment:
if we're micro-managing this line anyway, maybe can change it to `static bool` instead of `bool static`, having static first that seems to be a more common order in our source